themes/bootstrap3/templates/myresearch/bulk-action-buttons.phtml

```diff
@@ -6,7 +6,7 @@
 <nav class="bulkActionButtons">
```

Comment:
.bulkActionButtons may have float or flex stylings that need to be broken for a wrap.

Reply:
This already wraps as expected -- and this template is coded consistently with `search/bulk-action-buttons.phtml`. So the real question is, why is the cart different, as noted above?